### DISCUSSION TITLE
Windows and privileged containers

### DISCUSSION THREAD

**Discussion by PierreRAFFA** [actions/actions-runner-controller#2020] [Questions]
Hi,
Iam currently facing an issue for the windows runners running in EKS as ARC will start the runners in privileged mode.
The parameter `dockerdWithinRunnerContainer` does not any impact as ARC will add privileged mode to either the runner pod or the sidecar container.
From your documentation:
```When using the default runner, jobs that use a container will run in docker. This necessitates privileged mode, either on the runner pod or the sidecar container```
Also, `dockerEnabled` does not change anything.
Is there a way to NOT run the runner pod in privileged mode, especially for Windows or if `dockerEnabled` is false ?
Thanks

**Comment by nascit** [comment]
Hello @PierreRAFFA 
I'm having the same issue in EKS as well
Did you end up finding out what was the issue?

**Reply by ghost** [reply]
@PierreRAFFA and @nascit same here, did either of you figure out a workaround?
